### DISCUSSION TITLE
HM-1500 inputs performance with 25V & 51V battery

### DISCUSSION THREAD

**Discussion by Manos1966** [hoylabs/OpenDTU-OnBattery#244] [General]
Sorry guys, I am new at github and not yet perfectly capable of finding my way around...
During a discussion last Saturday, I offered to provide details of the performance of a Hoymiles HM-1500 connected to a 25V and 51V battery. For some strange reason, I can not find the post any more.
Since it more people may ask the same question, I post here the results.
This is a **Test Setup** trying to evaluate how to best achieve the soon-to-be allowable limit of "800 Watt Balkonkraftwerk Einspeisung". Earlier tests showed that the HM-800 (two inputs) with a 25V battery will not deliver more than 2x(25Vx10A) = 500Watt so we started experimenting different configurations (of course, we burned a few things on the way).

- Setup 
HOYMILES HM-1500 (4 inputs - 2 MPTT).
VICTRON MPPT 150/45 does 12 / 24 / 48 V (Auto Select) with four 405Wp panels.
Two "Power Queen 25,6V 100Ah LiFePO4, Eingebautes 100A BMS" Batteries in parallel, (later serial) configuration, delivering 25,6V (and later 51V).

The HM-1500 has four inputs, each capable of handling up to 11,5A. 
This means, currents over 11.5A will not be used by the inverter so,  if current is at 13A, only the 11,5A will be used, the remaining 1,5A (13-11,5) will not be used by the HOYMILES. 
Absolute Max is 15A per input that is, if current goes above 15A, you have a damaged HM-1500!
![HM1000-1200-1500 Eingang](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/cfda26dc-9230-47f6-b3be-7c358d915d28)

Be warned, as per Hoymiles, **the inputs are not made to be connected with batteries**. Doing so, may damage your inverter and will not be covered by the warranty so, any tests are done at your own risk!

- **With battery configuration parallel (25,6V)**
HM-1500 set at 100%. All four inputs connected to the battery: four cables (6qmm) from the negative pole of the battery go to the four negative inputs of the HM-1500, four cables (6qmm) from the positive pole of the battery go to the four positive inputs of the HM-1500.
Each input accepted approx. 9,8A at 25V delivering 242Watt (each of the four Inputs delivered between 234 - 242W)
Total for the HM-1500 was also 903Watt, much less than the 1500Watt max performance. Efficiency level 95%
Very good to see that the HM-1500 regulated itself at approx 9,8A per input (and did not go up in smoke...). 👍
![HM-1500 on 4 Input 25V Battery](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/d9f23fae-a3fa-4e17-8e1e-9705a0c83857)
-  **With battery configuration serial (51,2V)**
HM-1500 set at 100%. All four inputs connected to the battery: four cables (6qmm) from the negative pole of the battery go to the four negative inputs of the HM-1500, four cables (6qmm) from the positive pole of the battery go to the four positive inputs of the HM-1500.
Each input accepted approx. 9A at 45V delivering 400Watt (each of the four Inputs delivered between 375 - 402W)
The HM-1500 achieved its 1475Watt max performance. Efficiency level 95% 💯 
![HM-1500 on 4 Input 51V Battery](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/d1fbfe83-8f1f-4775-bae9-bad508097973)
As I understand (I maybe wrong, as I know very little about Solar & Batteries) due to the heavy load, the voltage of the batteries dropped from 51V down to 45V, this is supposed to be "normal"

On a side note, the VICTRON 150/45 switched (automatically) to 48V and is charging the batteries without problem - I thought as the batteries should have 51V, charging them with 48V VICTRON would be a problem - it does not look like it.
**UPDATE:** Connecting only two solar panels to the VICTRON does not provide sufficient voltage to charge fully the 51V battery. It seems you must have at least **three panels** connected to the VICTRON in order to achieve sufficient Voltage  to charge the 51V battery to max. Our first configuration was with four solar panels connected to the VICTRON, there we had no problems at all to charge the battery to the max.

**Comment by Manos1966** [comment]
Just adding to the above, VICTRON MPPT 150/45 does 12 / 24 / 48 V, but as the picture shows, delivers 54,34V at 21,50A, enough to charge the 51V battery (my respect for VICTRON increased tremendously).
![Leistung VICTRON](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/1dc844d9-b86b-4e8d-9a80-6dcd67c3f9b1)

**Comment by Rosetteonfire92** [comment]
> Sorry guys, I am new at github and not yet perfectly capable of finding my way around... During a discussion last Saturday, I offered to provide details of the performance of a Hoymiles HM-1500 connected to a 25V and 51V battery. For some strange reason, I can not find the post any more. Since it more people may ask the same question, I post here the results.
> 
>     * Setup
>       HM-1500
>       VICTRON MPPT 150/45 does 12 / 24 / 48 V (Auto Select) with four 405Wp panels.
>       Two "Power Queen 25,6V 100Ah LiFePO4, Eingebautes 100A BMS" Batteries in parallel and later serial configuration (25,6 and later 51V)
> 
> 
> The HM-1500 has four inputs, each capable of handling up to 11,5A. Over 11.5A can not be used. Absolute Max is 15A i.e. if current is at 13A, only 11,5A will be used. If current goes above 15A, you have a damaged HM-1500! ![HM1000-1200-1500 Eingang](https://user-images.githubusercontent.com/129660360/241055695-cfda26dc-9230-47f6-b3be-7c358d915d28.JPG)
> 
> Be warned, as per Hoymiles, **the inputs are not made to be connected with batteries**. Doing so, may damage your inverter and will not be covered by the warranty so, any tests are done at your own risk!
> 
>     * With battery configuration parallel (25,6V)
>       HM-1500 set at 100%. All four inputs connected to the battery.
>       Each input accepted approx. 9,8A at 25V delivering 242Watt (each of the four Inputs delivered between 234 - 242W)
>       Total for the HM-1500 was also 903Watt, much less than the 1500Watt max performance. Efficiency level 95%
>       Very good to see that the HM-1500 regulated itself at approx 9,8A per input (and did not go up in smoke...). 👍
>       ![HM-1500 on 4 Input 25V Battery](https://user-images.githubusercontent.com/129660360/241059453-d9f23fae-a3fa-4e17-8e1e-9705a0c83857.jpg)
> 
>     * With battery configuration serial (51,2V)
>       HM-1500 set at 100%. All four inputs connected to the battery.
>       Each input accepted approx. 9A at 45V delivering 400Watt (each of the four Inputs delivered between 375 - 402W)
>       The HM-1500 achieved its 1475Watt max performance. Efficiency level 95% 💯
>       ![HM-1500 on 4 Input 51V Battery](https://user-images.githubusercontent.com/129660360/241060611-d1fbfe83-8f1f-4775-bae9-bad508097973.jpg)
>       As I understand (I maybe wrong, as I know very little about Solar & Batteries) due to the heavy load, the voltage of the batteries dropped from 51V down to 45V, this is supposed to be "normal"
> 
> 
> On a side note, the VICTRON 150/45 switched (automatically) to 48V and is charging the batteries without problem - I thought as the batteries should have 51V, charging them with 48V VICTRON would be a problem - it does not look like it.

Kannst du mir für das Verständnis eine kurze Verkabelung darstellen und habe ich das richtig verinnerlicht, dass alle Solarmodule am WR und am Laderegler angeschlossen(parallel) sind? Wurde da separat abgesichert ? Ist das MPP-Tracking am WR und am Laderegler an ? Der angenommene Strom von 9,8 A bezieht sich auf die Kombination mit deinen 405er Modulen, oder? (Meine liefern schon deutlich mehr als 10 A..).

**Reply by Manos1966** [reply]
@Rosetteonfire92 I am sorry for the late reply, I had written the answer but it seems it was not posted...

"alle Solarmodule am WR und am Laderegler angeschlossen(parallel) sind? "
Nein, alle Solarmodule sind am VICTRON
Am HOYMILES (4 Eingaenge) sind 4 Mal Kabel von der Batterie

"Wurde da separat abgesichert ? "
Nein

"Ist das MPP-Tracking am WR und am Laderegler an ? "
Nein, nur am VICTRON

"Der angenommene Strom von 9,8 A bezieht sich auf die Kombination mit deinen 405er Modulen, oder? (Meine liefern schon deutlich mehr als 10 A..)."
Meine 405er are rated 12,96A STC and 10,39 A  NMOT
Der Wechselrichter scheint zu regeln auf 9,8 A pro Eingang,  unabhaengig ob 25V Battery, 51V Battery, HM-800, oder HM-1500.

**Comment by Manos1966** [comment]
One more information,
We are testing the 51V Battery setup.
Connecting only two solar panels to the VICTRON does not provide sufficient voltage to charge the 51V battery fully.
It seems one needs **min three panels** connected to the VICTRON if you want to charge the 51V battery efficiently
(with four panels, we had no problems at all).
Below the ratings for our test panels:
![405Panels](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/f19a1c2a-8a21-4227-954a-125da9eed299)

**Comment by Manos1966** [comment]
I noticed something interesting, the battery is empty from the night consumption (and not enough charging due to only two solar panels connected to VICTRON).
Pushed to the limit, voltage dropped rapidly (I presume, before OpenDTU had time to react to it) and since, the efficiency of the HOYMILES dropped from 95% down to 52% percent.
I manually lower max production from 1500W down to 50W (to allow VICTRON charge the battery) but, efficiency remains at 52%
The only way to bring efficiency back to the normal 95% is by manually restarting the HOYMILES. 
![Wirkung 52% A](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/652d7698-8754-4058-bcee-64d48d5f4cbb)
![Wirkung 52% B](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/c1ddbac6-7a16-45a2-9177-e75cbfbe7415)
![Wirkung 52% C](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/746018e0-3cc4-4ccf-a00c-6e4fe5d200ed)

I also have the suspicion, the challenge is caused by the VICTRON: It is charging the battery thus, delivers over 50V 
The battery is empty, HOYMILES tries to deliver xx Watt, empty battery voltage drops immediately below DC cutoff threshold, OpenDTU stops production - so far everything runs perfect! 
**However**,  as the VICTRON is still producing (and HOYMILES is set to null), battery voltage immediately jumps to 50V and higher (the VICTRON is also connected to the battery as well as the HOYMILES). So, DC start threshold is reached and the OpenDTU gives the order to HOYMILES to start producing again, and the cycle restarts. "Made by design" :-)
NOTE: This happens now, because the VICTRON is connected to **two 405W panels only** thus it would never be capable to deliver the required amount of energy.  IF all four solar panels were connected (1600w peak) and producing, and IF we would be limited to the 600W (soon to be 800W) limit, the problem would probably not appear! 👍 

I need to find a battery with CAN bus to test how it would perform....

PS. I am not criticising, I am just sharing my observations here. If you are not interested, you do not need to read this thread.

**Reply by helgeerbe** [reply]
> PS. I am not criticising, I am just sharing my observations here. If you are not interested, you do not need to read this thread.

We will never be able to test all possible setups. So every feedback is highly appreciated, which might help others.

**Reply by spcqike** [reply]
what are your settings in opendtu-onbattery? how are you reading battery voltage? have you connected your victron to your opendtu (ve.direct)? do you have a smartshunt? either from victron or a self build? (https://github.com/kscholty/SmartShuntINA226, i dont know if anyone ever tested this DIY smartshunt with this project, but says it uses VE.direct protocol.)

**Reply by Manos1966** [reply]
Thanks @helgeerbe , 

I also should mention, these setups are done "off-grid", in an open-space area at a construction site (Betriebsgelände), without flammable materials around. The construction site is producing over 130kW (solar+conventional) thus a HM-1500 is no issue to them.
DEFINITIVELY not tests one should do at home (at least I would not advice to do so, without having read twice your insurance contract).

**If** I needed 1500W for my house, I would have paid some 4K Euros, bought a big hybrid-inverter and have an electrician lay new lines...
It is very much the same as with charging your electric car at home using the standard electricity plug (Schuckostecker): Charging at 2700W is **possible but not advisable**. Yes, your hot-water kettle also consumes 2500W but, only for a period of a few minutes. Different   story when you draw 2700W continuously for six hours. Modern houses with new cabling can sustain it, older houses (older cabling) surely not!

Besides, if your house consumes more than 300W at night when you sleep, you are doing something wrong (unless if you have three servers running, as I do..).

**Comment by helgeerbe** [comment]
Can you share your Power Limiter settings?

@MalteSchm is working on a PR #232 which allows a battery precharge, before Solar Passthrough kicks in. Maybe this this will avoid the sudden voltage drops.

Just as a test:
You could try to set the stop threshold higher. So battery has a higher load if it is started again. Does this prevent the sudden voltage drops?

**Comment by Manos1966** [comment]
@spcqike
what are your settings in opendtu-onbattery? (see picture) 
how are you reading battery voltage? (same input string the Power Limiter is reading from)
have you connected your victron to your opendtu (ve.direct)? (yes)
do you have a smartshunt? (no, thanks for the tip, something to try in the future **although** I am thinking more about a test using a battery with CANbus

@helgeerbe 
**Question:** "Pylontech Battery support (via CAN bus interface)." Are there any other CANbus interface batteries the OpenDTU can support right now (because this is a difficult model to test right now).
Can you share your Power Limiter settings? (see picture)
The Hoymiles is still "blocked" at 52% efficiency and I have lowered the max limit from 1500 to 50 in order to charge the battery
I had not restarted it yet (now, after restart the HOYMILES is back to its 95%)
You could try to set the stop threshold higher. The issue is that the battery is empty because using the VICTRON with only two panels (instead of four) does not bring the voltage high enough to do full charge of a 51V battery! This was the most important information we needed - keep in mind, this is a VICTRON MPPT 150/45 12/24/48 V, we did not even expect it would be able to charge a 51V battery...
The basic principle is that the battery is used **at night* - The VICTRON is not producing, thus the only voltage comes from the battery - so, no problems ;-) 
We are not looking for the Eierlegendewohlmilchsau here.... I am just trying to understand the functionality and logic - and how it will behave in winter when there will be 15 hours of night instead of five in summer.  
![Wirkung 52% D](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/a9c98d0a-e03a-4850-93fc-6bdc63a3891f)

By the way, just mentioning: I get a strange error every xx minutes, I presume it is because there are three DTUs connected to the HOYMILES ;-) but just in case I mention it:
Everything zeroes, strings 1 & 2 show 2886,4Volt. Probably it is HOYMILES' polite way to tell me to "go to hell"
![Error 2886V](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/2cbae021-370e-469d-9b8d-bf664f389dda)

**Comment by Manos1966** [comment]
@spcqike asked the right question! 
I measure the battery voltage at the HOYMILES string!
Two 25V batteries, the BMS in each one will protect it from undervoltage. 
Unlikely they both stop at the same time so, when the first battery shuts down, voltage drops from 51-47V down to 24V. 
This explains (probably) the sudden drops.

**Comment by MalteSchm** [comment]
A couple of points:

- I think you should try and set the low voltage limit (DC stop voltage) to a higher value. If you see sudden voltage drops I think you're pushing it to hard. Certainly at the beginning when bringing this up.
- if you want to make sure that the battery is charged: Disable the Dynamic Power Limiter in the UI. Don't try and set the inverter to low power limits
- I would not expect that the Hoymiles operates perfect at these low power levels. 
- If the Victron MPPT does not charge your Batttery: Are you sure you configured the correct voltage limits in the MPPT?

**Reply by Manos1966** [reply]
@MalteSchm good questions but let's be realistic:
- Not many 2,5kWh Batteries , will "behave" if you start discharging them at 1,5kWh...  
  (and you must reduce capacity by 20% (10% off from the top and 10% off from the bottom), in order to keep the battery healthy a long time)
I am actually happy to see the Power Queen handle it without melting!
- Agree with you, the fact that the Hoymiles still works is beyond expectations 🥇 
- You ask the wrong VICTRON question, the right one should be: How can a MPTT rated at 48V max, charge a 51V battery? (because according to the forums, it should not be possible).

The OpenDTUonBattery also works beyond expectations, Respekt! 🥇 
The VICTRON is also amazing as well. Not only it delivers higher than its 48V rating, connected to 4 x 405Wp panels it delivers up to 1800W max (which should also be impossible).

However, the real issue we are doing this test is different: We are experimenting in order to see how much energy can somebody **realistically** use during the year (without costing him "an arm and a leg" as the current rates are around 30k EUR for a PV installation)?
![NAchtstundne p Monat](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/613b00d9-8055-4f00-a08c-8f6946a67ab6)
because during the winter months, the night hours are much longer (see picture) AND the solar power is lower and shorter.

**Reply by spcqike** [reply]
>     * You ask the wrong VICTRON question, the right one should be: How can a MPTT rated at 48V max, charge a 51V battery? (because according to the forums, it should not be possible).

well, the manual says it does.
https://www.victronenergy.com/upload/documents/Manual_SmartSolar_MPPT_75-10_up_to_100-20/29694-MPPT_solar_charger_manual-pdf-de.pdf
![grafik](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/50098985/4466a5af-1b77-4372-9bed-53b8a9b1ae85)

**Reply by Manos1966** [reply]
Yes but these are the recommended settings for the 48V system, not for a 51,2V system
![Victron 48V](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/01acfc5e-1de3-43b1-a49d-00b30063c695)

**Reply by spcqike** [reply]
Yes but as you use 2x25.6V, you have a 48V system. Havent you?

**Reply by Manos1966** [reply]
@spcqike This is a test-trap to test my multiplication skills, right ? ;-d
48V systems used to have 48V (2 x 24V Batteries).
Now that the 25V for Solar become fashion, things are getting complicated.
VICTRON has been manufacturing systems for 12V and 24V batteries for over 30years, I am sure they will update soon their systems to better handle the 25V series batteries (I want to believe that it is only a SW adjustment).

The difference in numbers look small but this is deceiving, for example:
The 48V system is setup for 56,8V, this makes an 8,8V difference
If you have a 51V  configuration, the same 56,8V from VICTRON make only 4,8V difference
This is **45% percent lower** compared to the differential the 48V system has. This is not a small difference.

**IF** (for example) this means it would require 45% longer to charge a 51V battery compared to a 48V one, we are "screwed" (in winter). A 25V system in such a case would be much better (smaller differential), even if the HM-1500 would not be able to deliver more than 4 x 25Vx10A = 1000W (50% percent less than its performance on a 51V system).

I do not have answers guys (I am not a battery guy), that is why we decided to test.

**Reply by spcqike** [reply]
Well, you know that a single lead cell has a nominal voltage of 2V, but can have a real voltage of 1.75V to 2.4V, depending on its charge state. That said a 12V battery can have 10.5V to 14.4V (as it has 6 cells) 

a 24V battery has 12 cells and a 48V battery has 24 cells, giving a total of 42V-57.6V, depending on its charging state. 

A single LiFePo4 cell has a nominal voltage of 3.2V, a minimal voltage of 3V and a maximum voltage of 3.4-3.45V. So you need either 4, 8 or 16 cells to get a 12/24/48V battery. In case of 16 cells you get a voltage range from 48V-55.2V with a nominal voltage of your mentioned 51.2V. 

So no. A „51.2V“ lifepo has a lower (maximum) voltage than a 48V lead acid battery. And in terms of 12/24/48V, it’s a 48V battery. Not a 51.2V one.

**Reply by Manos1966** [reply]
Many thanks for the explanations @spcqike this is really helpful, now I understand why the VICTRON works 👍 
Your comments tick a box in the list (we do not need to test whether a 25V vs a 51V configuration has a better efficiency with VICTRON).

**Reply by Manos1966** [reply]
@spcqike 

> In case of 16 cells you get a voltage range from 48V-55.2V

which also means I should put a **DC Spannung - Stop:** at 48,5V instead of 47V, right? 
And this increase **_could_** by itself solve the behaviour problem I experienced today.

**Comment by MalteSchm** [comment]
I guess I'm confused....  I understand now that there is no issue to solve. 
If you wonder about the 48V please read on rated vs. maximum voltage of lithium batteries. 

I'm not sure how I should connect some of the other statements made here (neither do I agree with all of them). But thats a different thing and I don't need to if there is no issue

**Reply by Manos1966** [reply]
Sorry for the confusion @MalteSchm 
This started as an informative post because somebody (at a different post) expressed fears that his HOYMILES HM-800 will be damaged by the high Amperes when it tries to achieve 800W using a 25V battery: 2 Stringsx25Vx16A = 800W but 16A is beyond the HOYMILES 15A max thus damage.
What I posted, shows that the Hoymiles HM-800, HM-1500 regulates the Amperes at 9-10A on a 25V and 50V battery. 

I do not have issues, the only thing I am trying to understand though is that, today around 12:00, the 50V (2 x 25,6V) battery voltage dropped all the way to 16V, **a behaviour it did not do during the previous days under similar _(we believe)_ conditions**.
On May 30th, it dropped to around 42V and recovered.
On May 31st, it handled the load quite nicely
Today June 1st, "crash and burn"
![Big Drop Voltage](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/a9b4bf9e-ca99-4dd4-a768-9edc94044701)
We did not change any settings as far as I can recall during these days. So that makes me wonder what was the reason for a different behavior.
As I have mentioned in previous posts, these are extreme conditions (we ask HOYMILES to deliver 1500W from the battery) which will not happen in real life as the official limit is 600W and soon 800W.
As I have the luxury to configure test the system I want before buying it, I prefer to know what I can expect (and what not).

**Comment by Manos1966** [comment]
Since many people spend their precious time to help me by giving suggestions and explanations, I would like to explain why I am testing:
An "average" person who will go buy a "Balkonkraftwerk" will soon want to buy a battery. Battery cost was very high two years ago, now prices dropped considerably to make them almost "affordable".
A typical 25V 100Ah 2.5kWh battery is at about 700EUR
If you want to keep the battery for a long time, you can not charge to to 100%, nor empty it completely.
Arbeite ich also im 15% - 90% Bereich, bleiben nur 1,88kWh die ich nutzen kann (siehe Bild).
Bei einem Strom Preis von 0,50 EUR pro kW und eine Entladung pro Tag, braeuchte ich 746 Tage um meine 700EUR Investition zurueck zu bekommen. Das sind fast genau zwei Jahre = es lohnt sich!
![Battery1](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/a19ba7f4-1d1e-43ed-8558-721644febefe)
... Voraussgesetz ich kann eine 1,88kWh Ladung pro Tag haben **und benutzen.**
Da wir unterschiedliche Dunkelheitstunden haben pro Monat, anbei die stundliche Watt Leistung, die diese Batterie liefern koennte durch die Nacht: 
![Battery2](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/9449d8f6-3f4c-4695-98e1-3039adf675a7)
Ausser der sommerlichen Zeit (May-August), ich behaupte jedes Haus hat einen naechtlichen Verbrauch von 110-150Watt, also es ist sicher, dass ich die volle Kapazitaet jedes Mal nutzen werde.
Amortisation binnen 2 Jahre, das macht die Sache interessant (dank OpenDTUonBattery).
**...Vorausgesetzt dass ich auch die Batterie auch jeden Tag voll bekomme**
Hier z.B. bei einen naechtlichen Verbrauch von 120W, und einer Ladeeffizienz von 90%, die Anzahl kWh die ich in die Batterie reinfliessen muss - dies wird zusaetzlich zu der taeglichen Solar-Hausstromversorgung gebraucht.  
![Battery3](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/718ba6c9-4dc1-4c1f-9136-50cb086725b0)
Da aber das Laden einer Batterie "off-Grid" passiert, gibt es keine 600W Einschraenkung mehr, ich kann so viele Panele und so viel Strom der VICTRON produzieren kann, ohne Einschraenkung nutzen.
Ich poste jetzt die Tabellen fuer die Kosten des Ladegeraets (VICTRON, HUAWEI usw) und der zusaetzlichen Panels nicht, eins ist aber sicher: So lange diese OpenDTUonBattery funktioniert, jeder Durschnitsverbraucher wird sich ein Balkonkraftwerk mit Batterie leisten koennen (und wird sein Geld zurueck sehen). 
Bevor ich ueber die OpenDTUonBattery gehoert hatte (sind ca. 2 Wochen her, ich bin selber AHOY User ;-) ), hielte ich dies fuer unmöglich!
Also, Respekt meine Herren! 🥇
